src/main/java/org/kik/bot/core/validator/XmlValidator.java

```diff
@@ -59,12 +61,11 @@ private boolean parseDataAsXml(String data) {
             dBuilder.parse(input);
             validXml = true;
         } catch (UnsupportedEncodingException | ParserConfigurationException e) {
-            e.printStackTrace();
+            LOGGER.error("Exception seen for " + data, e);
```

Comment:
I think as a default if the exception is provided as the second argument in the logger, it would display the class of error it that was passed. Specifying the class of error in the message might be redundant.

Reply:
@iservecharlie - apologies for the delay. 
Your suggestion makes sense. 
Let me change out the log message. 
For now, keeping it the same message `Exception seen for ` for the three catch clauses. 
If you want to merge them in, I can do that in a successive PR.